src/Command/ResetPasswordCommand.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Contao\Config;
+use Contao\Encryption;
+use Patchwork\Utf8;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * Resets password for a Contao backend user.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+class ResetPasswordCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('contao:reset-password')
```

Comment:
> Or maybe we call it contao:user:password because we also want to add a contao:user:create and possibly the same for members?

👍 

Regarding the command line option for the password: I agree we should keep it but we should maybe add a warning if the user uses it.